src/pudl/metadata/codes.py

```diff
@@ -53,6 +134,424 @@
         },
         "ignored_codes": [],
     },
+    "envr_equipment_manufacturer_codes_eia": {
```

Comment:
Wacky that they decided to use codes for the manufacturers instead of names.

Reply:
Indeed. We could decide to just skip the codes and replace them with the manufacturer name directly, might be more usable.